azure/microservice-ecommerce/blueprint.yaml

```diff
@@ -0,0 +1,201 @@
+    type: Input
+    description: What do you want to name the application?
+
+  - name: ProvisionCluster
+    type: Confirm
+    description: Do you want to provision a new Azure AKS cluster?
+
+  # ############################################################################
+  # Own cluster
+  # ############################################################################
+  - name: ClusterEndpoint
+    type: Input
+    description: What is the endpoint for the existing cluster, example https://104.155.109.230
+    dependsOnTrue: !expression "!ProvisionCluster"
+
+  - name: PasswordToken
+    type: Input
+    description: What is the token for accessing the existing cluster
+    dependsOnTrue: !expression "!ProvisionCluster"
+    secret: true
+  # ############################################################################
+
+  # ############################################################################
+  # New cluster
+  # ############################################################################
+  - name: AKSRegion
+    type: Select
+    description: "Select the AKS region for the cluster:"
+    dependsOnTrue: ProvisionCluster
+    options:
+    - eastasia
+    - southeastasia
+    - centralus
+    - eastus
+    - eastus2
+    - westus
+    - northcentralus
+    - southcentralus
+    - northeurope
+    - westeurope
+    - japanwest
+    - japaneast
+    - brazilsouth
+    - australiaeast
+    - australiasoutheast
+    - southindia
+    - centralindia
+    - westindia
+    - canadacentral
+    - canadaeast
+    - uksouth
+    - ukwest
+    - westcentralus
+    - westus2
+    - koreacentral
+    - koreasouth
+    - francecentral
+    - francesouth
+    - australiacentral
+    - australiacentral2
+    - southafricanorth
+    - southafricawest
+
+    # ############################################################################
+  # Authentication
+  # ############################################################################
+  - name: SubscriptionID
+    type: Input
+    description: "Enter you Azure subscription id:"
+    dependsOnTrue: ProvisionCluster
+
+  - name: TenantID
+    type: Input
+    description: "Enter your Azure tenant id:"
+    dependsOnTrue: ProvisionCluster
+
+  - name: ClientID
+    type: Input
+    description: "Enter your Azure client id:"
+    dependsOnTrue: ProvisionCluster
+
+  - name: ClientSecret
+    type: Input
+    description: "Enter your Azure client secret:"
+    secret: true
+    dependsOnTrue: ProvisionCluster
+
+  # ############################################################################
+  # For the Terraform backend storage
+  # ############################################################################
+  - name: ResourceGroup
+    # Resource group names only allow alphanumeric characters, periods, underscores, hyphens and parenthesis and cannot end in a period
+    type: Input
+    description: "Enter a new name for the Resource Group that will be created to house this cluster:"
```

Comment:
Can we rephrase the question as this gives an impression that we will create a Resource group when executing the blueprint

Reply:
But the blueprint does create a new Resource Group to house the cluster